### DISCUSSION TITLE
Development priorities

### DISCUSSION THREAD

**Discussion by NameOfTheDragon** [supermerill/SuperSlicer#4444] [General]
I've sponsored this project for a long time, even while @supermerill was AWOL before he quit his job to work on SS full time and nothing was happening. I'm a big fan, I love the project and I use it as my "daily driver". However, I'm starting to grow concerned that even with a full-time developer now, SS is falling too far behind Prusa Slicer. I want to keep using it, it is the darling slicer of the Voron community and as the owner of a pair of Vorons, I want that to continue to be true. But the grass is starting to look awfully green over in Prusa Slicer land. Organic supports. Text embossing tool. Cut tool. Measurement tool. And so on.

We've been getting 2.5 releases since March 2023. Is it now time to move on from 2.5 and focus on bringing in all the new goodness from Prusa Slicer 2.8?

If nothing else, please can we have tree/organic supports as a priority?

**Comment by krautech** [comment]
I am beginning to wonder this too. I love SS but some features thatre missing leave me using OrcaSlicer from time to time. Whilst I havent fully commited to OS yet, its getting that way.

Is there any news at all on development?

**Comment by dewi-ny-je** [comment]
> I've sponsored this project for a long time, even while @supermerill was AWOL before he quit his job to work on SS full time and nothing was happening. I'm a big fan, I love the project and I use it as my "daily driver". However, I'm starting to grow concerned that even with a full-time developer now, SS is falling too far behind Prusa Slicer. I want to keep using it, it is the darling slicer of the Voron community and as the owner of a pair of Vorons, I want that to continue to be true. But the grass is starting to look awfully green over in Prusa Slicer land. Organic supports. Text embossing tool. Cut tool. Measurement tool. And so on.
> 
> We've been getting 2.5 releases since March 2023. Is it now time to move on from 2.5 and focus on bringing in all the new goodness from Prusa Slicer 2.8?

Check this discussion to see if you maybe should use a different branch
https://github.com/supermerill/SuperSlicer/discussions/4090

**Comment by CuredPrusa** [comment]
I have tested the quality of print, Orca vs Super.
Super won.
Plus, a bunch of speed settings and cooling settings, abundant modifiers, it's hard to use anything else when I'm building a complex technical product.
But if bugs are left hanging in the air and I have to use three versions to complement each other in stability and functionality, it's getting tedious. 

In latest version of Super, I have noticed minor improvement in line layout, but all additional functions have bugs and freeze.  
E.g.  I have to use 2.5.59.13 to set internal bridges fan and then open that project to continue in 2.5.60.0 because I prefer better computing performance of newer version.  I feel like flying a Millennium Falcon : it flies fast and nicely, but you have to know it and find workarounds.
Most recenty, I have lost all overhangs in my project and can't retrieve them with any settings combination. I started to quote Roger Murtaugh  -I'm to old for this.... 

Still, when I open Prusa, I feel like I have disabled advanced mode (three dots in right corner) in Super.

I wish someone could tell me if I should switch to another slicer if Super is abandoned.

**Reply by DoubleStrike** [reply]
It's not abandoned - he is actively merging 2.7 now.  But, he has stated that for whatever reason, he cannot or will not provide periodic updates or estimates as to completion of that merge.  I don't understand why, but it's not my place to complain as he is making the updates.  I've just been pulling his branch and testing.
